client/lib/abtest/active-tests.js

```diff
@@ -61,4 +61,12 @@ module.exports = {
 		},
 		defaultVariation: 'notOffered'
 	},
+	monthlyPlanPricing: {
+		datestamp: '20160118',
```

Comment:
I don't have a strong opinion about it but it seems it makes sense since otherwise people might wonder why they can't trigger an A/B test despite following the testing instructions correctly. We encountered this very issue yesterday (p1452799515000182-slack-triage). @apeatling proposed to document this practice.